frontend/packages/ux-editor/src/classes/SavableFormLayoutSettings.test.ts

```diff
@@ -0,0 +1,21 @@
+import { SavableFormLayoutSettings } from '@altinn/ux-editor/classes/SavableFormLayoutSettings';
+import { formLayoutSettingsMock } from '../testing/mocks';
+import { FormLayoutSettings } from '@altinn/ux-editor/classes/FormLayoutSettings';
+
+describe('SavableFormLayoutSettings', () => {
+  const saveFormLayoutSettings = jest.fn();
+  const layoutSettings = new FormLayoutSettings(formLayoutSettingsMock);
+  const setupLayoutSettings = (): SavableFormLayoutSettings => {
+    return new SavableFormLayoutSettings(layoutSettings, saveFormLayoutSettings);
+  };
+
+  it('saves layoutSettings when pdfLayoutName is updated', () => {
+    const savableLayoutSettings = setupLayoutSettings();
+    const newPdfLayoutName = 'newPdfLayoutName';
+    savableLayoutSettings.setPdfLayoutName(newPdfLayoutName);
+    expect(saveFormLayoutSettings).toHaveBeenCalledTimes(1);
+    expect(saveFormLayoutSettings).toHaveBeenCalledWith(savableLayoutSettings.getLayoutSettings());
+    const actualNewPdfLayoutSetName = savableLayoutSettings.getPdfLayoutName();
+    expect(actualNewPdfLayoutSetName).toBe(newPdfLayoutName);
+  });
```

Comment:
Denne ville jeg også wrappet i en `describe('setPdfLayoutName', ...)` for å tydeliggjøre at det er denne funksjonen som blir testet.